core/control-plane/control-plane-contract/src/main/java/org/eclipse/edc/connector/controlplane/contract/validation/ContractValidationServiceImpl.java

```diff
@@ -40,7 +40,7 @@
 import java.util.Optional;
 
 import static java.lang.String.format;
-import static org.eclipse.edc.connector.controlplane.contract.spi.offer.ContractDefinitionResolver.CATALOGING_SCOPE;
+import static org.eclipse.edc.connector.controlplane.contract.ContractCoreExtension.CATALOG_SCOPE;
```

Comment:
this creates a dependency onto an extension class from a service class. We should avoid that, if possible, e.g. by moving the constant into an SPI class, or a `XYZConstants` class

Reply:
we don't have a convention on where to put `@PolicyScope`s, apparently they can be found both in extensions, service interfaces, and service implementations.

Given that they are (or should be) subject to automatic documentation generation, like for the `@Setting` annotations they seem to fit somewhat good in the extension class, but I don't have a strong opinion here.

Let's eventually discuss offline and tackle those in a dedicated issue/PR maybe